app/views/coronavirus_landing_page/components/shared/_section.html.erb

```diff
@@ -11,6 +11,12 @@
             href: item["url"],
             text: item["label"]
           }%>
+          <p class="govuk-body covid__list-item-description--featured">
+            <%=
+              featured_link_content_item = ContentItem.find!(item["url"])
+              featured_link_content_item.description
```

Comment:
I'd suggest using a helper method here. I'd add a helper at `app/helpers/content_item_helper.rb`:
```ruby
module ContentItemHelper
  def content_item_description(item)
    content_item = ContentItem.find!(item['url'])
    content_item.description
  end
end
```
Then here I'd just have:
```ruby
<%= content_item_description item %>
```

Oh and I'm assuming `item['description']` doesn't return the description - that is the description isn't already in the `item` hash